src/Symfony/Component/HttpFoundation/Session/Storage/Handler/RedisSessionHandler.php

```diff
@@ -45,7 +46,8 @@ public function __construct($redis, array $options = array())
             !$redis instanceof \RedisArray &&
             !$redis instanceof \RedisCluster &&
             !$redis instanceof \Predis\Client &&
-            !$redis instanceof RedisProxy
+            !$redis instanceof RedisProxy &&
+            !$redis instanceof RedisClusterProxy
```

Comment:
the Lock component also needs a similar patch

Reply:
👍 Done